src/routes/+page.svelte

```diff
@@ -1,11 +1,120 @@
 <script lang="ts">
-	import { Hiking } from 'svelte-google-materialdesign-icons';
+	let currentSide: string = '';
+	let leftLength: number = 300;
+	let rightLength: number = 300;
+	let mainContainer: HTMLElement;
```

Comment:
You don't need a reference to `mainContainer`, it should be possible to register the handlers on `document`, `window` or the bar itself. This is not something you need to change, just a suggestion :)

Reply:
We did this on purpose so that when clicking outside of the main container, it didn't do the stuffs

Reply:
Resolved as executive decision by Bastian and Anders